blockchain.go

```diff
@@ -592,6 +603,20 @@ func configureTransactionValidator(conf config, blocks *blocks) *access.Transact
 	)
 }
 
+func (b *Blockchain) newFvmContextFromHeader(header *flowgo.Header) fvm.Context {
```

Comment:
```suggestion
func (b *Blockchain) newFVMContextFromHeader(header *flowgo.Header) fvm.Context {
```

Reply:
Renamed in 230f39716dbbf4881379170235e520fb9d7a9e0f